.github/workflows/ci.yml

```
@@ -1,69 +1,177 @@
+        run: poetry run pytest src/tests/test_integration/ -n auto --cov=. --cov-append --cov-report=xml
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v5
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
+          slug: iSevenpwnz/Online-Cinema-Project
+
+  test-e2e:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)